include/termcolor/termcolor.hpp

```diff
@@ -202,6 +202,41 @@ namespace termcolor
         return stream;
     }
 
+    template <int code>
```

Comment:
As far as I understand, this manipulator uses any color from a 256-color palette. Can I ask you to use `uint8_t` instead of `int` then?